airbyte-config/config-models/src/main/resources/types/JobDiscoverCatalogConfig.yaml

```diff
@@ -8,10 +8,22 @@ additionalProperties: false
 required:
   - connectionConfiguration
   - dockerImage
+  - sourceId
+  - connectorVersion
+  - configHash
 properties:
   connectionConfiguration:
     description: Integration specific blob. Must be a valid JSON string.
     type: object
     existingJavaType: com.fasterxml.jackson.databind.JsonNode
   dockerImage:
     type: string
+  sourceId:
+    description: The ID of the source being discovered, so we can persist the result
```

Comment:
in all the following, can we add to the description these fields are required in order to persist the result?